pkg/ingester/ingester.go

```diff
@@ -1129,6 +1128,7 @@ func (i *Ingester) Push(ctx context.Context, req *cortexpb.WriteRequest) (*corte
 		if errors.As(firstPartialErr, &ve) {
 			code = ve.code
 		}
+		level.Warn(logutil.WithContext(ctx, i.logger)).Log("msg", "partial failure to push", "err", firstPartialErr)
```

Comment:
I think, since we move the logging to request level, it may be nice to provide some summary and rename some key fields like: 

```
msg: partial failures to push
failedSamples: 100
firstError: firstPartialErr
```

Reply:
Makes sense. I tried to add some useful info